### ISSUE TITLE
Missing field in the DiscussionNotePositionTextSchema

### ISSUE THREAD

**Issue by harjotgill** [jdalrymple/gitbeaker#3567]
**Description**

Missing `line_range` field in the `position` [schema](https://docs.gitlab.com/ee/api/discussions.html#list-project-merge-request-discussion-items) for DiscussionNotePositionTextSchema.

- Node.js version: N/A
- Gitbeaker version: 40.0.2
- Gitbeaker release (cli, rest, core, requester-utils): 40.0.2
- OS & version: N/A

**Steps to reproduce**

**Expected behaviour**

**Actual behaviour**

**Possible fixes**

**Checklist**

- [X] I have checked that this is not a duplicate issue.
- [X] I have read the documentation.

**Comment by harjotgill** [comment]
This is what I came up with based on what I see in the Webhook event for the DiffNote:

```typescript
export type DiscussionNotePositionTextSchema = DiscussionNotePositionBaseSchema & {
  position_type: 'text';
  new_line?: string;
  old_line?: string;
  line_range?: DiscussionNotePositionLineRangeSchema;
};

export interface DiscussionNotePositionLineRangeSchema {
  start?: DiscussionNotePositionTextLineSchema;
  end?: DiscussionNotePositionTextLineSchema;
}

export interface DiscussionNotePositionTextLineSchema {
  line_code?: string;
  type?: 'new' | 'old';
  old_line?: number;
  new_line?: number;
}
```

**Comment by jdalrymple** [comment]
:rocket: Issue was released in [`40.0.3`](https://github.com/jdalrymple/gitbeaker/releases/tag/40.0.3) :rocket:
